HW4_Vaganova/protein_analysis_tool.py

```diff
@@ -95,7 +119,21 @@ def find_site(seq: str, site: str) -> str:
     :param site: specify site of interest as short seq in 1-latter code (str)
     :return: positions of residues for each certain site in seq (str)
     """
-    pass
+    if seq not in RESIDUES_NAMES.values():
```

Comment:
This not appropriate type of site validation - here you are testing not site seq, but whole seq. Also you try to find all seq in dict values, however it is better way checking by letters in this seq.